README.md

```diff
@@ -1853,6 +1853,17 @@ class Twitter::API < Grape::API
 end
 ```
 
+Grape can also be told to rescue from all exceptions and still use the built in exception handing.
+This will give the same behavior as ```rescue_from :all``` with the addition that Grape will use the exception handling defined by all Exeption classes that inherit Grape::Exceptions::Base
```

Comment:
You don't need three back ticks on `rescue_from :all`. 

This is a full sentence so needs a period.